README.md

```diff
@@ -15,7 +15,7 @@ This library implements SSDP as defined in the
 and uses HTTP over Multicast UDP (HTTPMU) as defined in the
 [IETF draft](https://tools.ietf.org/html/draft-goland-http-udp-01).
 As an alternative, some devices use DNS-Based Service Discovery (DNS-SD) instead
-as defined in [RFC 6763](http://tools.ietf.org/html/rfc6763).
+as defined in [RFC 6763](https://tools.ietf.org/html/rfc6763).
```

Comment:
```suggestion
as defined in [RFC 6763](https://www.rfc-editor.org/rfc/rfc6763.html).
```

Reply:
After a small research I would agree to this changes and would commit them here additionally. Thanks for looking into this @clue 🥳